src/Tasks/Microsoft.NET.Build.Tasks/GenerateBundle.cs

```diff
@@ -18,13 +20,24 @@ public class GenerateBundle : TaskBase
         public bool IncludeSymbols { get; set; }
         [Required]
         public string OutputDir { get; set; }
+        [Required]
+        public string TargetFrameworkVersion { get; set; }
+        [Required]
+        public string RuntimeIdentifier { get; set; }
+
         [Required]
         public bool ShowDiagnosticOutput { get; set; }
 
+        [Output]
+        public ITaskItem[] ExcludedFiles { get; set; }
+
         protected override void ExecuteCore()
         {
+            OSPlatform targetOS = RuntimeIdentifier.StartsWith("win") ? OSPlatform.Windows :
```

Comment:
Should this be walking the RuntimeIdentifier graph instead of checking what the RuntimeIdentifier starts with?